src/main/resources/README.txt

```diff
@@ -5,5 +5,55 @@
 |__/|__/_/_/ /_/\__,_/\__,_/ .___/ 
                           /_/      
 
-Windup ${project.version} is a tool used to simplify Java application migrations.
+Windup ${project.version} is an extensible and customizable rule-based tool that helps simplify migration of Java applications.is an extensible and customizable rule-based tool that helps simplify migration of Java applications.
```

Comment:
A typo: 
"applications.is an extensible and customizable rule-based tool that helps simplify migration of Java applications."